src/app/shared/object-collection/shared/listable-object/listable-object.decorator.spec.ts

```diff
@@ -80,4 +100,44 @@ describe('ListableObject decorator function', () => {
       });
     });
   });
+
+  describe('With theme extensions', () => {
+    describe('If requested theme has no match', () => {
+      beforeEach(() => {
+        environment.themes = [
+          { name: 'requested', extends: 'intermediate' },
+          { name: 'intermediate', extends: 'ancestor' },
+        ];
+      });
+
+      it('should return component from ancestor theme if it has a match', () => {
```

Comment:
I have the same questions about this `describe`->`it`. 

In other words, the sentence..."If requested theme has no match...it 'should return component from ancestor theme if it has a match'" doesn't make sense to me.  This appears to be trying to test for extending themes, but I'm not understanding the test description.

I again think some inline comments could be helpful here...it's hard to follow what `typeHier1` versus `typeHier2` means, and why they should return different components.

Reply:
Tried to reword the `it`s & explain a bit better